draft-irtf-cfrg-opaque.md

```diff
@@ -398,145 +398,331 @@ This specification defines one particular AKE based on 3DH; see {{instantiations
 We let `Npk` and `Nsk` denote the size of public and private keys, respectively,
 used in the AKE.
 
-# Offline Registration {#offline-phase}
+- Deterministic Key Pair Generation Function:
+  - DeriveKeyPair(seed): Derive a private and public key pair deterministically from the seed.
+  - HashToScalar(msg, dst): Hash arbitrary input to a scalar in the field of the AKE group as defined in {{I-D.irtf-cfrg-voprf}}, except with dst = "OPAQUE-HashToScalar".
```

Comment:
```suggestion
```

This isn't an API that's exposed outside of its use in DeriveKeyPair, right? If not, we can probably drop it from here, and describe it where we specify how to implement DeriveKeyPair. (In the configurations section?)